core/src/main/java/org/stellar/anchor/util/AssetHelper.java

```diff
@@ -20,7 +20,7 @@ public static boolean isISO4217(String assetCode, String assetIssuer) {
     }
   }
 
-  public static boolean isStellar(String assetCode, String assetIssuer) {
+  public static boolean isIssuedAsset(String assetCode, String assetIssuer) {
```

Comment:
```suggestion
  public static boolean isNonNativeAsset(String assetCode, String assetIssuer) {
```

Reply:
I feel the name of the function isn't very clear. It would be helpful to add some comments here.

Reply:
Changed the name and added Javadoc